src/components/VideoChatButtonAndMenu.js

```diff
@@ -100,8 +98,8 @@ class VideoChatButtonAndMenu extends Component {
                     <Pressable
                         onPress={() => {
                             // If this is the Concierge chat, we'll open the modal for requesting a setup call instead
-                            if (this.props.openInboxCall) {
-                                Navigation.navigate(ROUTES.getRequestCallRoute(this.props.taskID));
+                            if (this.props.isConcierge) {
+                                Navigation.navigate(ROUTES.getRequestCallRoute('NewExpensifyConciergeDM'));
```

Comment:
umm whats `NewExpensifyConciergeDM`? i don't see this on web-e master.
Don't you mean `ExpensifyCashConciergeDM`? Or is there a PR i'm unaware of that's updating this text?

Reply:
That was updated in this PR https://github.com/Expensify/App/pull/4408

Reply:
I don't get it. But where is the new taskID defined in web-e. Like shouldn't web-E be aware of this taskID?
In web-E i see [this](https://github.com/Expensify/Web-Expensify/blob/20364a8874680e0d9485c1b9a7e725f46003a0c5/lib/Expensiworks/InboxCallUser.php#L64) which uses `ExpensifyCashConciergeDM`. So if we pass `NewExpensifyConciergeDM` then its treated as false [here](https://github.com/Expensify/Web-Expensify/blob/20364a8874680e0d9485c1b9a7e725f46003a0c5/lib/Expensiworks/InboxCallUser.php#L204) which feels incorrect. Thoughts? Am i missing something?

Reply:
No I don't think you are missing something, I think that was missed in the other updates PR. I'll spin up a separate issue for that.

Reply:
PR: https://github.com/Expensify/Web-Expensify/pull/31749
Follow up issue to remove the old task ID from web: https://github.com/Expensify/Expensify/issues/174159

Reply:
So does this PR now go on HOLD till the web-E code is on production or not really?

Reply:
I think that since this is a n6 issue, we should avoid HOLDing this. We can CP the other PR to help with backwards compatability